lib/tests/streamlit/file_util_test.py

```diff
@@ -72,7 +72,7 @@ def test_streamlit_write(self):
         ) as makedirs, file_util.streamlit_write(FILENAME) as output:
             output.write("some data")
             open().write.assert_called_once_with("some data")
-            makedirs.assert_called_once_with(dirname)
+            makedirs.assert_called_once_with(dirname, exist_ok=True)
```

Comment:
`streamlit.util.os.makedirs` appears twice in this file, and seems like an anti-pattern. Should we fix to just `os.makedirs`?

Reply:
Yeah, probably

Reply:
@tvst  I see stuff like `streamlit.util.os.makedirs` all over the tests.  Can we talk about why it's there?

Reply:
Talked to Thiago; will try to convert these, but if it's not an easy win i'm not going try to get this done in this particular PR.

Reply:
Intermittent problems happen when I change these.  I'm gonna change them back.

```test_multiple_scriptrunners - tests.streamlit.scriptrunner.ScriptRunner_test.ScriptRunnerTest
tests/streamlit/scriptrunner/ScriptRunner_test.py
AssertionError: Lists differ: ['False', 'ahoy!', '2', 'False', 'loop_forever'] != []  First list contains 5 additional elements. First extra element 0: 'False'  - ['False', 'ahoy!', '2', 'False', 'loop_forever'] + []
self = <scriptrunner.ScriptRunner_test.ScriptRunnerTest testMethod=test_multiple_scriptrunners>

    def test_multiple_scriptrunners(self):
        """Tests that multiple scriptrunners can run simultaneously."""
        # This scriptrunner will run in parallel to the other 3.
        # It's used to retrieve the widget id before initializing deltas on other runners.
        # Wait a beat to access deltas.
        scriptrunner = TestScriptRunner("widgets_script.py")
        scriptrunner.enqueue_rerun()
        scriptrunner.start()
        time.sleep(0.1)
    
        # Build several runners. Each will set a different int value for
        # its radio button.
        runners = []
        for ii in range(3):
            runner = TestScriptRunner("widgets_script.py")
            runners.append(runner)
    
            states = WidgetStates()
            wid = scriptrunner.get_widget_id("radio", "radio")
            _create_widget(wid, states).int_value = ii
            runner.enqueue_rerun(widget_state=states)
    
        # Start the runners and wait a beat.
        for runner in runners:
            runner.start()
    
        time.sleep(0.1)
    
        # Ensure that each runner's radio value is as expected.
        for ii, runner in enumerate(runners):
            self._assert_text_deltas(
>               runner, ["False", "ahoy!", "%s" % ii, "False", "loop_forever"]
            )

ii         = 2
runner     = <scriptrunner.ScriptRunner_test.TestScriptRunner object at 0x7fd3cc2b2a20>
runners    = [<scriptrunner.ScriptRunner_test.TestScriptRunner object at 0x7fd3cc2b2c18>, <scriptrunner.ScriptRunner_test.TestScriptRunner object at 0x7fd3cc2b2c88>, <scriptrunner.ScriptRunner_test.TestScriptRunner object at 0x7fd3cc2b2a20>]
scriptrunner = <scriptrunner.ScriptRunner_test.TestScriptRunner object at 0x7fd3cc2b1e48>
self       = <scriptrunner.ScriptRunner_test.ScriptRunnerTest testMethod=test_multiple_scriptrunners>
states     = widgets {
  id: "4838589110670875929"
  int_value: 2
}

wid        = '4838589110670875929'

tests/streamlit/scriptrunner/ScriptRunner_test.py:293: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
tests/streamlit/scriptrunner/ScriptRunner_test.py:367: in _assert_text_deltas
    for delta in scriptrunner.deltas()
E   AssertionError: Lists differ: ['False', 'ahoy!', '2', 'False', 'loop_forever'] != []
E   
E   First list contains 5 additional elements.
E   First extra element 0:
E   'False'
E   
E   - ['False', 'ahoy!', '2', 'False', 'loop_forever']
E   + []```